src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -13,6 +13,9 @@
     <Platforms>x64;x86;arm;armv6;arm64;riscv64;s390x;wasm;ppc64le</Platforms>
 
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+
+    <!-- CA2252: Opt in to preview features before using them (Lock) -->
```

Comment:
Should it rather be in the shared `src\libraries\System.Private.CoreLib\src\System.Private.CoreLib.Shared.projitems`?

Reply:
That would work better, updated